test/OptionSettlementEngine.t.sol

```diff
@@ -129,9 +129,38 @@ contract OptionSettlementTest is Test, NFTreceiver {
         assertEq(engine.balanceOf(BOB, testOptionId), 0);
     }
 
-    function testWriteMultipleWriteSameOptionType() public {
-        IOptionSettlementEngine.OptionLotClaim memory claim;
+    function testClaimAccessor() public {
+        vm.startPrank(ALICE);
+        uint256 claimId1 = engine.write(testOptionId, 69);
+        IOptionSettlementEngine.OptionLotClaim memory claimLot = engine.claim(claimId1);
+
+        assertEq(claimLot.amountExercised, 0);
+        assertEq(claimLot.amountWritten, 69);
+        assertEq(claimLot.optionId, testOptionId);
+        assertTrue(claimLot.unclaimed);
+
+        vm.warp(testExerciseTimestamp + 1);
 
+        // Alice would probably never exercise her own option irl
+        engine.exercise(testOptionId, 1);
```

Comment:
Test feedback
- Worth it imo, in this test or another, to transfer some options to BOB and have him exercise
- Probably worth it to do one more iteration in this test, where you make a second exercise on the lot and again assert values are updated, including unclaimed/unredeemed

Reply:
yeah, there are already plenty of tests where BOB exercises, was just going for brevity here.

good fb on exercise 2